gradle/libs.versions.toml

```diff
@@ -134,15 +116,25 @@ spring-data-commons = { module = "org.springframework.data:spring-data-commons",
 
 # JDBC
 
-micronaut-sql-jdbc-tomcat = { module = "io.micronaut.sql:micronaut-jdbc-tomcat", version.ref = "micronaut-sql" }
-micronaut-sql-jdbc = { module = "io.micronaut.sql:micronaut-jdbc", version.ref = "micronaut-sql" }
-
 drivers-jdbc-oracle = { module = "com.oracle.database.jdbc:ojdbc8", version.ref = "oracle-jdbc-driver" }
 flyway-oracle = { module = "org.flywaydb:flyway-database-oracle", version.ref = "oracle-flyway" }
```

Comment:
@radovanradic Can you please add this dependency here https://github.com/micronaut-projects/micronaut-flyway/blob/master/gradle/libs.versions.toml so we can access from the Flyway BOM

Reply:
Please take a look https://github.com/micronaut-projects/micronaut-flyway/pull/402